File: Hafta1/problem setleri/mario-less.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height; 
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);  

    for (int i = 0; i < height; i++) 
    {
        for (int j = height - i - 1; j > 0; j--) 
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++) 
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Hafta 6/pset6/mario_less.py
from cs50 import get_int

while True:
    h = get_int('height: ')
    if h < 1 or h > 8:
        h = get_int('height: ')
    if h >= 1 or h <= 8:
        break

for i in range(h):
    print((h - 1 - i) * " ", end="")
    print((i + 1) * "#")